pom.xml

```diff
@@ -208,6 +193,19 @@ under the License.
     </dependency>
   </dependencies>
 
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <id>plexus.snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/plexus-snapshots</url>
+    </repository>
+  </repositories>
```

Comment:
I am wondering if this correct? Should we have another repository directly hardcoded in the POM?

I know this has been used to get SNAPSHOTS in the past, but currently the plexus-compiler is a release version, and I assume it should stay that way.